pkg/flare/archive.go

```diff
@@ -35,10 +35,13 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 
 	"github.com/mholt/archiver"
-	yaml "gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v2"
 )
 
 const (
+	heapProfileName = "heap_profile.log"
```

Comment:
This is also a `.profile ` and not a log

Reply:
Made it a `.log` because `go-routine-dump` is one as well, but yes, it would be more readable as `.profile`. I will change it.